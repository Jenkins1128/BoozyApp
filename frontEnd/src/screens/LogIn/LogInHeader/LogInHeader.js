import React from 'react';
import { View, Image, ActivityIndicator, StyleSheet } from 'react-native';
import logo from '../../../images/logo.png';

const LogInHeader = ({ state }) => {
	return (
		<View style={styles.header}>
			<Image source={logo} style={styles.logoImage}></Image>
			{state.isLoading && <ActivityIndicator style={styles.loadingIndicator} size='large' color='#fff' />}
		</View>
	);
};

const styles = StyleSheet.create({
	loadingIndicator: {
		marginTop: 20
	},
	header: {
		flex: 2,
		alignItems: 'center',
		marginTop: 100
	},
	logoImage: {
		width: '30%',
		height: '40%',
		opacity: 0.7
	}
});

export default LogInHeader;
