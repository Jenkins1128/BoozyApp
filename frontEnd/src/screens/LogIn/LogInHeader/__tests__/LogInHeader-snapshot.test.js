import React from 'react';
import renderer from 'react-test-renderer';
import LogInHeader from '../LogInHeader';

test('LogInHeader renders correctly', () => {
	const state = {
		isLoading: false
	};
	const tree = renderer.create(<LogInHeader state={state} />).toJSON();
	expect(tree).toMatchSnapshot();
});
