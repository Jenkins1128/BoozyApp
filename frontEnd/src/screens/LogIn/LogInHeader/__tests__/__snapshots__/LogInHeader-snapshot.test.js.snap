// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogInHeader renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 2,
      "marginTop": 100,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/images/logo.png",
      }
    }
    style={
      Object {
        "height": "40%",
        "opacity": 0.7,
        "width": "30%",
      }
    }
  />
</View>
`;
